### PR TITLE
Relocation

### PR DESCRIPTION
**Pull request by hbrunn**
Updated version of #95

### CODE REVIEW

File: anybox/recipe/odoo/base.py [hunk 1]
```diff
@@ -1618,6 +1629,17 @@ def finalize_addons_paths(self, check_existence=True):
                 assert os.path.isdir(path), (
                     "Not a directory: %r (aborting)" % path)
 
+        self.addons_paths = [
+            os.path.relpath(
+                addons_path,
+                os.path.join(
+                    self.bin_dir,
+                    'openerp' if 'openerp' in addons_path else 'odoo'
+                ))
```
Comment (thomaspaulb) [inline anybox/recipe/odoo/base.py]: In my configuration this goes wrong: 

- `self.bin_dir` is `$BUILDOUT/bin`
- `addons_path` is `$BUILDOUT/parts/some_repo` and as such does not contain either `openerp` or `odoo`.
- Total expression comes to `$BUILDOUT/bin/odoo`
- Relative path becomes `../../parts/some_repo` and is written to `$BUILDOUT/etc/odoo.cfg`
- Odoo is run from `$BUILDOUT/parts/odoo/openerp`, tries to find addons in $BUILDOUT/parts/parts/some_repo`, fails.

Comment (hbrunn) [reply anybox/recipe/odoo/base.py]: @sebastiken seems your patch doesn't work for everyone, you you have a look at https://github.com/anybox/anybox.recipe.odoo/pull/104#pullrequestreview-63599099?

Comment (sebastiken) [reply anybox/recipe/odoo/base.py]: Ok @hbrunn I will see this using the configuration proposed by @thomaspaulb

**Comment by StefanRijnhart** [comment]
I'm interested in merging this feature. We have a version that has run on production for years now. Is it alright if we propose that instead of this conflicting PR?

**Comment by hbrunn** [comment]
@StefanRijnhart please

**Pull request closed without merging**